README.md

```diff
@@ -1,17 +1,30 @@
 # Morpho Token
 
-This repository contains the Morpho protocol’s ERC20 token.
+This repository contains the Morpho protocol's ERC20 token.
 It is designed to be upgradable and support onchain delegation.
 Additionally, it ships a wrapper contract to simplify the migration of assets from the previous token contract to the new Morpho token contract.
 
-## Upgradability
+## Features
 
-The Morpho token complies with the [EIP-1967](https://eips.ethereum.org/EIPS/eip-1967) to support upgradeability.
+### Upgradeability
 
-## Delegation
+The Morpho token complies with [EIP-1967](https://eips.ethereum.org/EIPS/eip-1967) to support upgradeability.
+
+### Delegation
 
 The Morpho token supports onchain voting and voting power delegation.
 
+### Role-based Permission
+
+The new Morpho token does not have role-based permission of functions.
+
+### Burning Tokens
+
+This version brings a breaking change for this feature.
+In the legacy Morpho token, burning tokens was made possible by transferring them to the zero address.
+This approach is deprecated in this version of the Morpho token.
+To burn tokens, approved users may call the `burn` function.
```

Comment:
I don't think this section is useful + it's adding legal risk. We've been strongly advised to keep feature parity between both the legacy and new tokens. Or at least I would remove the "breaking change" expression.

Reply:
Noted. It's worth mentioning that wording suggests that not that feature support is broken but rather the way it's implemented. I understand your concern. I'll let you guys decide as I lack of a clear view to favor one choice or another.

Reply:
I removed the "breaking change" mention, and changed the rest a bit